data/globalevents/scripts/startup.lua

```diff
@@ -42,4 +41,16 @@ function onStartup()
 		local position = town:getTemplePosition()
 		db.query("INSERT INTO `towns` (`id`, `name`, `posx`, `posy`, `posz`) VALUES (" .. town:getId() .. ", " .. db.escapeString(town:getName()) .. ", " .. position.x .. ", " .. position.y .. ", " .. position.z .. ")")
 	end
+
+	if configManager.getBoolean(configKeys.CHECK_DUPLICATES_VALUES) then
+		-- List of table names to be checked for duplicates
+		local variableNames = {"AccountStorageKeys", "PlayerStorageKeys", "GlobalStorageKeys", "actionIds", "uniqueIds"}
+		-- Loop through the list of table names
+		for _, variableName in ipairs(variableNames) do
+			-- Call the checkDuplicatesValues function for each table
+			local hasDuplicates, message = checkDuplicatesValues(variableName)
+			-- Print the result of the check for each table
+			print(">> Checking " .. variableName .. ": " .. message)
+		end
+	end
```

Comment:
Hmm I believe duplicate storage values to be a critical error (it will lead to bugs, 100% sure) so it can crash the server and prevent it from starting. If the server owner wants to use duplicate values they can always disable the check, so if enabled, it should prevent invalid server state.

Reply:
I don't know how to shut down the server, because `Game.setGameState(GAME_STATE_CLOSED)` didn't work in this case.

Reply:
Hmm, leave it as is for now, I don't think we have a "shutdown" function in Lua